### PR TITLE
ISSUE-302 Documentation for rate limiting features and its routes

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

File: docs/modules/ROOT/pages/tmail-backend/webadmin.adoc [hunk 1]
```diff
@@ -131,4 +131,305 @@ Returns the list of Team Mailboxes `bob@domain.tld` user has access to :
...
+	"transitLimits": [{
+			"name": "receivedMailsPerHour",
+			"periodInSeconds": 3600,
+			"count": 100,
+			"size": 2048
+		},
+		{
+			"name": "receivedMailsPerDay",
+			"periodInSeconds": 86400,
+			"count": 1000,
```
Comment (chibenwa) [inline docs/modules/ROOT/pages/tmail-backend/webadmin.adoc:186]: I think we don't need to return the plan that was created... A 201 CREATED should be enough...

File: docs/modules/ROOT/pages/tmail-backend/features/tmailRateLimiting.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
+= TMail Rate Limiting
+:navtitle: TMail Rate Limiting
+
+Manage email rate limiting plans and apply to users!
```
Comment (chibenwa) [inline docs/modules/ROOT/pages/tmail-backend/features/tmailRateLimiting.adoc:4]:
```suggestion
Manage email rate limiting plans and apply to them to your users!
```

File: docs/modules/ROOT/pages/tmail-backend/features/index.adoc [hunk 3]
```diff
@@ -31,4 +31,16 @@ Team mailboxes enable everybody in the team:
 
 An example would be `marketting@linagora.com`.
 
-xref:tmail-backend/features/teamMailboxes.adoc[Read more]...
\ No newline at end of file
+xref:tmail-backend/features/teamMailboxes.adoc[Read more]...
+
+== TMail Rate Limiting
+
+Manage email rate limiting plans and apply to users!
```
Comment (chibenwa) [inline docs/modules/ROOT/pages/tmail-backend/features/index.adoc:38]:
```suggestion
Manage email rate limiting plans and apply to them to your users!
```

File: docs/modules/ROOT/pages/tmail-backend/webadmin.adoc (hunk 1 above)
Comment (quantranhong1999) [reply docs/modules/ROOT/pages/tmail-backend/webadmin.adoc]: I think we can return just planId.

**Comment by chibenwa** [comment]
Looks overall good to me

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
Can you squash?

**Pull request merged**